src/components/application_manager/src/commands/mobile/system_request.cc

```diff
@@ -466,6 +466,13 @@ void SystemRequest::Run() {
     file_name = kSYNC;
   }
 
+  std::size_t backslash_index = file_name.find('\\');
+  if (backslash_index != std::string::npos) {
+    LOG4CXX_ERROR(logger_, "File name contains backslash");
+    SendResponse(false, mobile_apis::Result::INVALID_DATA);
```

Comment:
@Kozoriz Well the other SendResponse() methods in "system_request" command are sending only 2 parameters, so to correspond with them I removed "info" parameter from my SendResponse() method.